src/test/regress/expected/multi_extension.out

```diff
@@ -26,6 +26,10 @@ DROP EXTENSION citus;
 \c
 -- Create extension in oldest version, test every upgrade step
 CREATE EXTENSION citus VERSION '5.0';
+-- test version control check, this should error out
+SELECT * FROM pg_dist_partition;
+ERROR:  citus binaries does not match installed extension
+DETAIL:  Please update installed extension with ALTER EXTENSION citus UPDATE;
 ALTER EXTENSION citus UPDATE TO '5.0-1';
```

Comment:
After we add the `ALTER EXTENSION` hook, this will fail.

@anarazel, what do you think about that, since you said there should be a failsafe to keep users from upgrading to anything except the current `so`'s supported version?